pkg/elemental/elemental_test.go

```diff
@@ -605,14 +600,23 @@ var _ = Describe("Elemental", Label("elemental"), func() {
 			Expect(err).ShouldNot(HaveOccurred())
 		})
 		It("Copies files from a directory source", func() {
-			sourceDir, err := utils.TempDir(fs, "", "elemental")
+			rsyncCount := 0
+			src := ""
+			dest := ""
+			runner.SideEffect = func(cmd string, args ...string) ([]byte, error) {
+				if cmd == "rsync" {
```

Comment:
Maybe worth comparing against the constant now

Reply:
arent you traveling? lol